vaccine_feed_ingest/runners/us/physicians_immediate/normalize.py

```diff
@@ -0,0 +1,181 @@
+    #           ('StreetName', 'Green Bay Road'),
+    #           ('StreetNamePostType', 'Mount'),
+    #           ('PlaceName', 'Pleasant'),
+    #           ('StateName', 'WI'),
+    #           ('ZipCode', '53406')]),
+    #        'Street Address'),
+    #
+    # The correct name of the town is "Mount Pleasant"
+    if (
+        address.get("StreetNamePostType") == "Mount"
+        and address.get("PlaceName") == "Pleasant"
+    ):
+        del address["StreetNamePostType"]
+        del address["PlaceName"]
+        address["PlaceName"] = "Mount Pleasant"
+
+    # Fixup:
+    #  (OrderedDict([('AddressNumber', '3111'),
+    #                ('StreetNamePreDirectional', 'S.'),
+    #                ('PlaceName', 'Chicago South Milwaukee'),
+    #                ('StateName', 'WI'),
+    #                ('ZipCode', '53172')]),
+    #  'Street Address'),
+    #
+    # 'Chicago' is a 'StreetName', located in the town of 'South Milwaukee'
+    if address.get("PlaceName") == "Chicago South Milwaukee":
+        del address["PlaceName"]
+        address["StreetName"] = "Chicago"
+        address["PlaceName"] = "South Milwaukee"
+
+    # Fixup:
+    # (OrderedDict([('AddressNumber', '2490'),
+    #               ('StreetName', 'Bushwood'),
+    #               ('PlaceName', 'Dr.Elgin'),
+    #               ('StateName', 'IL')]),
+    # 'Street Address'),
+    #
+    # 'Dr.Elgin' is a typographical error. Create a 'StreetNamePostType' of 'Dr.' and a new 'PlaceName' of 'Elgin'
+    if address.get("PlaceName") == "Dr.Elgin":
+        del address["PlaceName"]
+        address["StreetNamePostType"] = "Dr."
+        address["PlaceName"] = "Elgin"
+
+    return address
+
+
+def parse_vaccine(site: Site) -> Optional[List[schema.Vaccine]]:
+    """Not all sites report what type of vaccine they carry, or report "-" when
+    they are out of supply. Every site reports at most one type of vaccine
+    """
+    # Bail early on sites with no vaccine info
+    if site.get("type") is None:
+        return None
+
+    if int(site["slots"]) > 0:
+        supply = schema.VaccineSupply.IN_STOCK
+    else:
+        supply = schema.VaccineSupply.OUT_OF_STOCK
+
+    potentials = {
+        "pfizer": schema.VaccineType.PFIZER_BIONTECH,  # not observed in the data
+        "moderna": schema.VaccineType.MODERNA,
+        "johnson & johnson": schema.VaccineType.JOHNSON_JOHNSON_JANSSEN,
+    }
+    match = potentials.get(site["type"].lower().strip())
+
+    if match is None:
+        return None
+    else:
+        return [{"vaccine": match, "supply_level": supply}]
+
+
+def normalize(site: Site, timestamp: str) -> schema.NormalizedLocation:
+    return schema.NormalizedLocation(
+        id=get_site_id(site),
+        name=site["clinic"],
+        address=infer_address(site),
+        inventory=parse_vaccine(site),
+        source=schema.Source(
+            source="us_physicians_immediate",
+            id=site["row_id"],
```

Comment:
Let's have this id match the overall `id`
```suggestion
            id=f"{site['table_id']}_{site['row_id']}",
```